packages/text-area/test/text-area.test.js

```diff
@@ -238,12 +243,16 @@ describe('text-area', () => {
       expect(textArea.clientHeight).to.be.below(height);
     });
 
-    it('should not change height', () => {
+    it('should not change height', async () => {
       textArea.style.maxHeight = '100px';
-      textArea.value = Array(400).join('400');
```

Comment:
Strange why it didn't work before.

Reply:
Somehow without this change it doesn't pass in Playwright, not sure about the reason.